parts/kubernetesagentcustomdata.yml

```diff
@@ -123,6 +123,7 @@ write_files:
             --cluster-domain=cluster.local \
             --cloud-provider=azure \
             --cloud-config=/etc/kubernetes/azure.json \
+            --azure-container-registry-config=/etc/kubernetes/azure.json \
```

Comment:
should this be added to the windows powershell (function Write-KubernetesStartFiles)